app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -51,22 +52,27 @@ class BindingTariffClassificationConnector @Inject()(appConfig: AppConfig, clien
     client.GET[Option[Case]](url)
   }
 
+  private def buildQueryUrl(types : Seq[ApplicationType] = Seq(ApplicationType.BTI,ApplicationType.LIABILITY_ORDER), withStatuses: String,
+                            queueId: String = "", assigneeId: String, pagination: Pagination): String = {
+    val sortBy = "application.type,application.status,days-elapsed"
+    val queryString = s"application_type=${types.mkString(",")}&queue_id=$queueId&assignee_id=$assigneeId&status=$withStatuses&sort_by=$sortBy&sort_direction=desc&page=${pagination.page}&page_size=${pagination.pageSize}"
+    val url = s"${appConfig.bindingTariffClassificationUrl}/cases?$queryString"
```

Comment:
this could be the param to return?  s"${appConfig.bindingTariffClassificationUrl}/cases?$queryString"

Reply:
ok, sure. Makes it a little more compact